### PR TITLE
docs(layouts): update layout docs

### LINKED ISSUE #317: Remove the base href from templates meant for digestion
## Describe the Bug
Some people (myself) may be too inept to know to remove the base href and spend way too much time trying to figure out what is wrong with my life. This seems to be in place exclusively to make it work in demos. If possible, it would be good to remove this just to avoid issues should someone not be familiar with that html bit and overlook it. 🤷‍♂️ 

## Steps to Reproduce
Use any of the templates. e.g. https://github.com/rackerlabs/helix-ui/blob/master/docs/components/layouts/vertical-layout-template.html

## Expected behavior
Change links to js/css and make it work

## Additional Context
Line in question: https://github.com/rackerlabs/helix-ui/blob/master/docs/components/layouts/vertical-layout-template.html#L8

### PR DESCRIPTION
**Pull request by CITguy**
JIRA: freefriday

* update Box classes in documentation layout
* update layout templates 
  * stop referencing `helix-ui.polyfills.js`
  * simplify consumption of `custom-elements-es5-adapter.js`
* tweak margins on `ul.hxList` and `.hxFigure`

Closes #317

### LGTM's
- [x] Dev LGTM
- [x] Info Dev LGTM
- [x] Zoom LGTM

### CODE REVIEW

**Comment by HelixUI** [comment]
Deploy preview for *helix-ui* processing.

Built with commit 105416f7f510e48adfe0283ffd5ef302e8d5abde

https://app.netlify.com/sites/helix-ui/deploys/5b89e38ce39e7c551fc3324f

File: docs/components/layouts/horizontal-layout.html [hunk 1]
```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en"><!-- lang is necessary for screen reader compatibility -->
+{% include 'partials/head.njk' %}
```
Comment (100stacks) [inline docs/components/layouts/horizontal-layout.html]: please add a title to the page.

File: docs/components/layouts/index.html [hunk 2]
```diff
@@ -10,225 +10,381 @@
...
-            <footer id="foot">...</footer>
-          </body>
-        </html>
+        <div id="stage">
+          <!-- (optional) Nav -->
+          <nav id="nav">
+            <!-- internal application links -->
+          </nav>
+
+          <!-- (optional) Left Siderail -->
```
Comment (100stacks) [inline docs/components/layouts/index.html]: siderail needs to be two words

File: docs/components/layouts/index.html [hunk 3]
```diff
@@ -10,225 +10,381 @@
...
+          <aside class="hxSiderail">
+            <!-- secondary page content -->
+          </aside>
+
+          <main id="content">
+            <!-- primary page content -->
+          </main>
+
+          <!-- (optional) Right Siderail -->
```
Comment (100stacks) [inline docs/components/layouts/index.html]: siderail needs to be two words

File: docs/components/layouts/index.html [hunk 4]
```diff
@@ -10,225 +10,381 @@
...
-            </div>
-            <footer id="foot">...</footer>
-          </body>
-        </html>
+        <nav id="nav">
+          <!-- links to pages within application -->
+        </nav>
       {% endcode %}
-      <figcaption>Abbreviated HTML markup for a horizontal application layout</figcaption>
+      <figcaption> HTML markup for the nav area</figcaption>
```
Comment (100stacks) [inline docs/components/layouts/index.html]: please remove extra space

File: docs/components/layouts/index.html [hunk 5]
```diff
@@ -10,225 +10,381 @@
...
-          support for accessible user agents.
-        </li>
-        <li>Give the element an id of <code>content</code>.</li>
-      </ul>
+    <ul>
+      <li>
+        It must have an ID of <code>content</code>.
+      </li>
+      <li>
+        Use a <code>&gt;main&gt;</code> element to provide the best support
```
Comment (100stacks) [inline docs/components/layouts/index.html]: please change the first `gt` to `lt`

File: docs/components/layouts/index.html [hunk 6]
```diff
@@ -10,225 +10,381 @@
...
+    <p>
+      The side rail houses secondary page content.
+    </p>
+    <ul>
+      <li>
+        Use an <code>&lt;aside&gt;</code> element to provide the best support
+        for accessible user agents.
+      </li>
+      <li>
+        Add a class of <code>hxSiderail</code> to apply visual styling.
```
Comment (100stacks) [inline docs/components/layouts/index.html]: clarify that this is a css class

File: docs/components/layouts/vertical-layout.html [hunk 7]
```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
```
Comment (100stacks) [inline docs/components/layouts/vertical-layout.html]: please add a title to this page.

File: docs/files/vertical-layout-template.html [hunk 8]
```diff
@@ -0,0 +1,79 @@
...
+  <a href="#content">Skip to main content</a>
+
+  <header id="head">
+    <!-- Eyebrow Goes Here -->
+  </header>
+
+  <div id="app">
+    <div id="stage">
+      <nav id="nav" class="hxNav">
+        <a href="#">Link 1</a>
```
Comment (100stacks) [inline docs/files/vertical-layout-template.html]: please add begin and end comments

**Comment by 100stacks** [comment]
Zoom LGTM

**Comment by chri2547** [comment]
LGTM!

**Pull request merged**